uber/models.py

```diff
@@ -1758,7 +1758,19 @@ def differences(cls, instance):
             new_val = getattr(instance, attr)
             old_val = instance.orig_value_of(attr)
             if old_val != new_val:
-                diff[attr] = "'{} -> {}'".format(cls.repr(column, old_val), cls.repr(column, new_val))
```

Comment:
A comment explaining why we do this would be worthwhile.  At a glance if I saw this I'd probably assume someone was being overly paranoid about thrown exceptions instead of realizing that there's a real-world "expected" case where this can happen (i.e. datatype-changing database migrations).